tests/unit/test_orderbook_rewards.py

```diff
@@ -45,7 +68,32 @@ def test_rewards_user_generated_view_query(self):
         """.strip().replace(
             "        ", ""
         )
-        self.assertEqual(expected, orderbook_rewards_query(period, self.example))
+        self.assertEqual(
+            expected, RewardQuery.AGGREGATE.dune_query(period, self.aggregate_example)
+        )
+
+    def test_per_order_user_generated_view_query(self):
```

Comment:
Same as above. This won't run for me because of this error:

```
FileNotFoundError: [Errno 2] No such file or directory: './queries/user_generated_per_order_rewards.sql'
```

Is this `.sql` file to be included in the git?

Reply:
I'm pretty sure it is. CI is passing. Remember that the relative imports need to be fixed.

Reply:
Same as above. Thanks!